packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -202,8 +207,19 @@ export class SerializedStateManager {
 	 * base snapshot when attaching.
 	 * @param snapshot - snapshot and blobs collected while attaching
 	 */
-	public setSnapshot(snapshot: SnapshotWithBlobs | undefined) {
-		this.snapshot = snapshot;
+	public setInitialSnapshot(snapshot: SnapshotWithBlobs | undefined) {
+		if (this.offlineLoadEnabled) {
+			assert(this.snapshot === undefined, "inital snapshot should only be defined once");
+			assert(snapshot !== undefined, "attachment snapshot should be defined");
+			const { baseSnapshot, snapshotBlobs } = snapshot;
+			const attributesHash =
+				".protocol" in baseSnapshot.trees
```

Comment:
```suggestion
isCombinedAppAndProtocolSummary
```

Reply:
That one takes an ISummaryTree and we have a ISnapshotTree

Reply:
ahh bummer